src/platforms/web/runtime/modules/style.ts

```diff
@@ -82,7 +82,7 @@ function updateStyle(oldVnode: VNodeWithData, vnode: VNodeWithData) {
   // to mutate it.
   vnode.data.normalizedStyle = isDef(style.__ob__) ? extend({}, style) : style
 
-  const newStyle = getStyle(vnode, true)
+  const newStyle = getStyle(vnode, true) as any
```

Comment:
instead of "as any" use can create proper types like whatever getStyles returns with or, | undefined.